src/System.Private.CoreLib/shared/System/Globalization/RegionInfo.cs

```diff
@@ -50,6 +48,8 @@ public RegionInfo(string name)
             }
 
             SetName(name);
+            // workaround compiler bug
```

Comment:
Are you guys considering a marker (like TODO or TODONULLABLE) to find all these later when we can fix them?

Reply:
I think this should be easy to find with simple analyzer since you assign self to self with `!`.

Just talked with folks and seems coreclr already has convention for TODOs so we will go ahead and add `TODO-NULLABLE` on all these.

Also I'm actually not 100% sure compiler team will be able to fix this except for simple cases since the flow analysis in some cases might be kinda hard

Reply:
ok, fixed with https://github.com/dotnet/coreclr/pull/23526/commits/0eeb85ce1febe56e530a96cbbd67dfc26f6257dc

Reply:
thanks!